### PR TITLE
Website iterator

### PR DESCRIPTION
**Pull request by Uriel867**
Added a unit test and a function in order to check if a given parameter is a URL or just text

### CODE REVIEW

File: training/training_population.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+               'et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut\n' \
+               'aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in\n' \
+               'voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat\n' \
+               'non proident, sunt in culpa qui officia deserunt mollit anim id est laborum\n' \
+               'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium,\n'
+
+        iterator = get_iterator_for_website(text)
+        result = iterator.next()
+        assert ['Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore',
+                'et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut',
```
Comment (smartFreshBear) [inline training/training_population.py:60]: @Uriel867 few things: 
1-  you *never* mix test with production code.
as I said before, please follow examples of other existing tests

2- your test has only very basic functionally.
I would expect testing edge cases as well (for example, test next after next, test edge cases like text with only 1 line... be creative and destructive ;)

**Review by smartFreshBear — CHANGES_REQUESTED** [review]
please see comments
and carefully see where test logic is located with respect to the module.

File: training/training_population.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        self.content_length = len(self.content)
+
+    def get_website_text(self):
+        all_text = ''
+        if self.is_website_url(self.url):
+            all_text, title = text_extractor.get_all_text_from_url(self.url)
+        else:
+            all_text = self.url
+        return all_text
+
```
Comment (smartFreshBear) [inline training/training_population.py:22]: this is a workaround, usually, we want a constructor to accept only one type.
this approach is confusing, I will accept it for now, may open a ticket to user "Factory Design Pattern" (welcome to read about)
also you could been avoid it if you will learn on Mocks - for next time.

**Review by smartFreshBear — APPROVED** [review]
Approved with comments
